net/unbound/files/unbound.init

```diff
@@ -6,6 +6,9 @@ START=61
 USE_PROCD=1
 
 start_service() {
+	chmod -R u=rwX,g=rX,o= /etc/unbound
+	chown -R unbound:unbound /etc/unbound
+
```

Comment:
Unfortunately not. The postinstall code can't meaningfully change ownership when a static image, i.e. with unbound built in, is made (building without root permissions and using `fakeroot`, of course).